### ISSUE TITLE
Docker:   403-forbidden on first access.   4443 ncp is completely incaccessable

### ISSUE THREAD

**Issue by rekcodocker** [nextcloud/nextcloudpi#1450]
I installed NextcloudPi on docker on a clean Raspberry Pi, to the letter. 
I started the container. 

I browse to htpps://my.domain.suffix/activate.

Result: 
HTTP 403 Forbidden, You don't have permission to access this resource.

I try to browse to htpps://my.domain.suffix:4443 to access ncp:  Connection refused.
I try to browse within my Lan to TheServerWhichHostsNextcloud.lan:4443 : Connection refused.

I found the terminal-based ncp-config. This is not stated in the manual. The manual simply implies that everything works when you run this single line of code to start the docker container.
Anyways. Nothing I do there helps.
There are certificate problems. Chrome is getting impossibly difficult with self-signed certificates. I had to use a different browser.
I can not run LetsEncrypt from the ncp-config terminal:  It too receives a 403 error.
I cannot set 'trusted domains' to *.*.*.* - it errors with Invalid Character. 

Basically, the docker-installation is locked, and there are no instructions on how to unlock it.
The instructions "start the docker container and browse to YourDomain/activate" simply are inadequate. Something is missing.
